src/components/member-card/index.js

```diff
@@ -45,17 +48,40 @@ const Card = ({ developerInfo, isOptionKey }) => {
       {dev && (
         <SuperUserOptions username={username} showSettings={showSettings} />
       )}
-      <motion.img
-        layoutId={username}
-        src={
-          isMember ? `${img_url}?${Math.random() * 100}` : '/images/Avatar.png'
-        }
-        onError={brokenImageHandler}
-        className={
-          isMember ? classNames.imgContainer : classNames.imgContainerNewMember
-        }
-        alt={username}
-      />
+
+      {isMember && (
+        <motion.img
+          layoutId={username}
+          src={
+            isMember
+              ? `${img_url}?${Math.random() * 100}`
+              : '/images/Avatar.png'
+          }
+          onError={brokenImageHandler}
+          className={
+            isMember
+              ? classNames.imgContainer
+              : classNames.imgContainerNewMember
```

Comment:
do we really need this check? given the fact this section only renders when `isMember` is true 🤔

Reply:
thanks for the suggestion, we don't need it